app/src/lib/stores/alive-store.ts

```diff
@@ -192,6 +192,11 @@ export class AliveStore {
   }
 
   private subscribeToAccount = async (account: Account) => {
+    // For now, Alive is only available for GitHub.com accounts
+    if (account.endpoint !== getDotComAPIEndpoint()) {
```

Comment:
Do you think we could add an endpoint capability function for this instead? My hope is that we'll move away from this pattern. Something like

```ts
export const supportsAliveSessions = endpointSatisfies({
  dotcom: true,
  ae: false,
  es: false,
})
```

In `endpoint-capabilities.ts` and then invoke that instead?

Reply:
True! Sorry, I already forgot about it, I'm the worst 😂 

Changed in b7653f1